### PR TITLE
Wallet: Create Account UI (2)

### LINKED ISSUE #17126: Wallet Create Account screen follow up
Dependencies: 

- https://github.com/status-im/status-mobile/issues/17124
- https://github.com/status-im/status-mobile/issues/17122
- https://github.com/status-im/status-mobile/issues/17120

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/17126

This is a PR is a follow up to https://github.com/status-im/status-mobile/pull/17118. It fixes some UI details that were pending/missing.

[Designs](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1-132380&mode=design&t=PJiI0P0LEHGBakdp-0)

Screenshot_20231002_092020_Status Debug

### CODE REVIEW

File: src/status_im2/contexts/wallet/create_account/view.cljs [hunk 1]
```diff
@@ -66,8 +66,10 @@
          (i18n/label :t/colour)]
         [quo/color-picker
          {:default-selected @account-color
-          :on-change        #(reset! account-color %)}]]
-       [rn/view {:style (style/divider-line theme)}]
+          :on-change        #(reset! account-color %)
+          :container-style {:padding-horizontal 12
```
Comment (ibrkhalil) [inline src/status_im2/contexts/wallet/create_account/view.cljs]: Kindly check [this](https://github.com/status-im/status-mobile/pull/17465) PR Omar, We don't want horizontal margins for color picker anymore as it adds bad UX

Comment (OmarBasem) [reply src/status_im2/contexts/wallet/create_account/view.cljs]: @ibrkhalil this is padding not a margin, and applied to the `content-container-style`. The color picker usage on different screens have different padding. The component is taking full device width.

File: src/quo2/components/colors/color_picker/view.cljs [hunk 2]
```diff
@@ -18,10 +18,11 @@
    - `blur?` Boolean to enable blur background support.}"
   [{:keys [default-selected]}]
   (let [selected (reagent/atom default-selected)]
-    (fn [{:keys [blur? on-change feng-shui?]}]
+    (fn [{:keys [blur? on-change feng-shui? container-style]}]
```
Comment (J-Son89) [inline src/quo2/components/colors/color_picker/view.cljs:21]: 👌

File: src/status_im2/contexts/wallet/common/temp.cljs [hunk 3]
```diff
@@ -79,8 +79,23 @@
...
+   {:title       (i18n/label :t/derivation-path)
+    :image       :icon
+    :image-props :i/derivated-path
+    :action       :button
+    :action-props {:on-press    #(js/alert "Button pressed!")
+                   :button-text (i18n/label :t/edit)
+                   :icon-left :i/placeholder
+                   :alignment    :flex-start}
+    :description :text
+    :description-props {:text "m / 44’ / 60’ / 0’ / 0’ / 2"}}])
```
Comment (J-Son89) [inline src/status_im2/contexts/wallet/common/temp.cljs]: the default derivation path exists in the codebase btw, it's in status-im2 constants,perhaps we should use that here?

**Review by J-Son89 — APPROVED** [review]

**Comment by OmarBasem** [comment]
@Francesca-G a design review please :)

**Review by Francesca-G — COMMENTED** [review]
Here's the Figma [frame](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6258%3A7665&mode=design&t=M1KtotHHcOFvU96f-1) with the design review :)

**Comment by OmarBasem** [comment]
@Francesca-G thanks for your review. Replied to your comments on Figma

**Comment by OmarBasem** [comment]
@Francesca-G fixed the button text

**Review by briansztamfater — APPROVED** [review]

File: src/quo2/components/settings/settings_item/style.cljs [hunk 4]
```diff
@@ -13,9 +13,10 @@
   {:flex-direction :row
    :align-items    (if (or tag description) :flex-start :center)})
 
-(def sub-container
+(defn sub-container
+  [align-action]
```
Comment (smohamedjavid) [inline src/quo2/components/settings/settings_item/style.cljs:17]: it's best to name this as `align-items`.

**Review by smohamedjavid — APPROVED** [review]

File: src/quo2/components/settings/settings_item/style.cljs (hunk 4 above)
Comment (OmarBasem) [reply src/quo2/components/settings/settings_item/style.cljs:17]: We are not aligning items here. This is to align the `action` component

**Review by Francesca-G — APPROVED** [review]

**Pull request merged**